{"kind":"symbol","schemaVersion":{"major":0,"patch":0,"minor":3},"abstract":[{"text":"Wrong cipher text size","type":"text"}],"hierarchy":{"paths":[["doc:\/\/SwiftKyber\/documentation\/SwiftKyber","doc:\/\/SwiftKyber\/documentation\/SwiftKyber\/KyberException"]]},"metadata":{"title":"KyberException.cipherTextSize(value:)","modules":[{"name":"SwiftKyber"}],"fragments":[{"kind":"keyword","text":"case"},{"kind":"text","text":" "},{"kind":"identifier","text":"cipherTextSize"},{"text":"(","kind":"text"},{"kind":"externalParam","text":"value"},{"text":": ","kind":"text"},{"text":"Int","kind":"typeIdentifier","preciseIdentifier":"s:Si"},{"kind":"text","text":")"}],"roleHeading":"Case","role":"symbol","externalID":"s:10SwiftKyber0B9ExceptionO14cipherTextSizeyACSi_tcACmF","symbolKind":"case"},"identifier":{"url":"doc:\/\/SwiftKyber\/documentation\/SwiftKyber\/KyberException\/cipherTextSize(value:)","interfaceLanguage":"swift"},"sections":[],"primaryContentSections":[{"declarations":[{"languages":["swift"],"platforms":["macOS"],"tokens":[{"kind":"keyword","text":"case"},{"kind":"text","text":" "},{"text":"cipherTextSize","kind":"identifier"},{"text":"(","kind":"text"},{"text":"value","kind":"externalParam"},{"kind":"text","text":": "},{"text":"Int","preciseIdentifier":"s:Si","kind":"typeIdentifier"},{"kind":"text","text":")"}]}],"kind":"declarations"}],"variants":[{"paths":["\/documentation\/swiftkyber\/kyberexception\/ciphertextsize(value:)"],"traits":[{"interfaceLanguage":"swift"}]}],"references":{"doc://SwiftKyber/documentation/SwiftKyber/KyberException/cipherTextSize(value:)":{"fragments":[{"text":"case","kind":"keyword"},{"text":" ","kind":"text"},{"kind":"identifier","text":"cipherTextSize"},{"text":"(","kind":"text"},{"kind":"externalParam","text":"value"},{"kind":"text","text":": "},{"text":"Int","kind":"typeIdentifier","preciseIdentifier":"s:Si"},{"text":")","kind":"text"}],"url":"\/documentation\/swiftkyber\/kyberexception\/ciphertextsize(value:)","kind":"symbol","role":"symbol","title":"KyberException.cipherTextSize(value:)","type":"topic","abstract":[{"text":"Wrong cipher text size","type":"text"}],"identifier":"doc:\/\/SwiftKyber\/documentation\/SwiftKyber\/KyberException\/cipherTextSize(value:)"},"doc://SwiftKyber/documentation/SwiftKyber":{"role":"collection","type":"topic","kind":"symbol","title":"SwiftKyber","identifier":"doc:\/\/SwiftKyber\/documentation\/SwiftKyber","abstract":[],"url":"\/documentation\/swiftkyber"},"doc://SwiftKyber/documentation/SwiftKyber/KyberException":{"fragments":[{"text":"enum","kind":"keyword"},{"kind":"text","text":" "},{"text":"KyberException","kind":"identifier"}],"url":"\/documentation\/swiftkyber\/kyberexception","kind":"symbol","role":"symbol","abstract":[{"text":"Kyber exceptions","type":"text"}],"title":"KyberException","type":"topic","identifier":"doc:\/\/SwiftKyber\/documentation\/SwiftKyber\/KyberException","navigatorTitle":[{"text":"KyberException","kind":"identifier"}]}}}